src/renderer/store.ts

```diff
@@ -1,19 +1,284 @@
-import { createStore, combineReducers } from 'redux';
```

Comment:
It's impressive how small this store is especially considering it has selectors and all!